dart/lib/src/noop_hub.dart

```diff
@@ -14,11 +14,11 @@ class NoOpHub implements Hub {
   }
 
   @override
-  void bindClient(SentryClient client) {}
+  void bindClient(SentryClient? client) {}
 
   @override
   Future<SentryId> captureEvent(
-    SentryEvent event, {
+    SentryEvent? event, {
```

Comment:
I guess this class is not properly checked, it differs from the real implementation

Reply:
Dang, why didn't the compiler complain about that? :(